awswrangler/catalog/_definitions.py

```diff
@@ -31,6 +31,7 @@ def _parquet_table_definition(
     table: str,
     path: str,
     columns_types: Dict[str, str],
+    table_type: Optional[str],
```

Comment:
Adding `table_type` argument to distinguish between Governed and non Governed tables